moveit_core/robot_state/test/test_cartesian_interpolator.cpp

```diff
@@ -14,7 +14,7 @@
  *     copyright notice, this list of conditions and the following
  *     disclaimer in the documentation and/or other materials provided
  *     with the distribution.
- *   * Neither the name of the Willow Garage nor the names of its
```

Comment:
Hmm, I'm not sure about changing the copyright holder on a preexisting file. I think this should be reverted.

Reply:
never mind, i see that you're just matching L4